BEST_PRACTICE.md

```diff
@@ -0,0 +1,247 @@
+# Clib Examples and Best Practice
+
+This page will cover:
+
+ - [How to use libraries](#how-to-use-installed-libraries).
+ - [Example Makefile](#example-makefile).
+ - [Example `package.json` for executables](#example-packagejson-for-exacutables).
+ - [Making your own library package](#making-your-own-libraries).
+ - [Example `package.json` for libraries](#example-packagejson-for-libraries).
+ - [How to install/uninstall executables](#install-and-uninstall-executables).
+
+For instructions on installation, click [here](https://github.com/clibs/clib#installation).
+
+## How to use installed libraries for your project
+
+Lets say you have your project, with a typical directory tree like this:
+
+```
+your-project/
```

Comment:
Please use the `tree` program to generate this. It looks much "nicer" 😄

Reply:
I did use the `tree` program, I just copy-pasted it. I dont know why it looks different here :thinking: